### ISSUE TITLE
MP4 Video playback freezes and crashes game

### ISSUE THREAD

**Issue by Heathenlamb** [YoYoGames/GameMaker-Bugs#8856]
### Description

MP4 Video playback freezes and crashes game. Whenever I load a video, it plays for a few seconds then freezes the game and crashes. Happens every time I run the game.

### Expected Change

For the video not to freeze and crash the game

### Steps To Reproduce

Load a video into the player when games starts.
Wait.
Game video freezes after a few seconds and crashes the game.

### How reliably can you recreate this issue using your steps above?

Always

### Which version of GMRT are you reporting this issue for?

2024.11.0.226

### Contact Us Package Attached?

- [ ] I have attached my Contact Us Package

### Sample Project Added?

- [x] I have included a small sample project

**Comment by Heathenlamb** [comment]
This is on the Windows platform.

**Comment by Heathenlamb** [comment]
Similar to this ticket -
https://github.com/YoYoGames/GameMaker-Bugs/issues/8451

**Comment by Heathenlamb** [comment]
I have export the project as best I can -
https://u.pcloud.link/publink/show?code=XZVYt55Z7jfdvs24JCm5AAwi8z0NFSaGu6Xk

I use GMS for making Jackpot tracking for clubs so the project files reference a whole bunch of external stuff. I think I've shut down enough of it to only have the video play and show what happening.

**Comment by jackerley** [comment]
The video in the attached project plays fine for me. What hardware are you running on? Are your video drivers up to date?

**Comment by Heathenlamb** [comment]
> The video in the attached project plays fine for me. What hardware are you running on? Are your video drivers up to date?

Im running it on an 12thgen intel with 128gb ram and a 3080 ti. Drivers all uptodate.

Ive run it on my home pc as well with the same issue ... dual xeon 24cores ..  64gb ram ... 3080ti ... all drivers up to date. 

H264 codecs all good. I do animation and video editing for a living so the PCs are doing what they have too.

Did you let it run for a good 10-20s ... sonetime it does it in 3 secs ... other times it takes 20-30s

**Comment by Heathenlamb** [comment]
For note I've been using Lonewolfs ultimate video player for GMS on Windows without a problem ... I only have this issue with the Native GMS player

**Comment by jackerley** [comment]
This issue is flagged for GMRT, are you actually using GMRT? Or are you using GMS2 VM\YYC? What runtime version are you using?

**Comment by Heathenlamb** [comment]
> This issue is flagged for GMRT, are you actually using GMRT? Or are you using GMS2 VM\YYC? What runtime version are you using?

Sorry I misunderstood ... I thought GMRT was a call all term for the GM Runtime in general. I am using GMS VM/YYC

**Comment by Heathenlamb** [comment]
The runtime is listed above - 2024.11.0.226

**Comment by Heathenlamb** [comment]
I bumped the close button ... that shouldnt be so easy haha

**Comment by alicemoretti** [comment]
Labels changed :)

**Comment by Heathenlamb** [comment]
> Labels changed :)

Thanks Alice !

**Comment by shichen85** [comment]
I can actually replicate the issue with runtime 2024.11.0.227. The game freezes after a few seconds and becomes unresponsive. Only by commenting out `game_set_speed()` at obj_control's creation event does the video play normally without freezing.
 
Not sure if the display's refresh rate might be a factor in this issue. Mine is 59.95 Hz.

**Comment by Heathenlamb** [comment]
> I can actually replicate the issue with runtime 2024.11.0.227. The game freezes after a few seconds and becomes unresponsive. Only by commenting out `game_set_speed()` at obj_control's creation event does the video play normally without freezing.
> 
> Not sure if the display's refresh rate might be a factor in this issue. Mine is 59.95 Hz. ![Image](https://github.com/user-attachments/assets/02ac0253-2d42-448c-b665-59af7a8311b6)

Yeah .. I have a feeling its connected to the ticket I linked above. It's been marked as "closed" ... but I think it's till an issue.
